libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -11,22 +11,36 @@
 
 [role="xpack"]
 [[monitoring-internal-collection]]
+
+
+[IMPORTANT]
+=========================
+Metricbeat is the recommended method for collecting and shipping monitoring data
```

Comment:
```suggestion
{metricbeat} is the recommended method for collecting and shipping monitoring data
```

Reply:
(good to use asciidoc attributes to resolve product names here...I'm trying to make this a cleanup step as we add or modify content)